modules/installation-azure-user-infra-uploading-rhcos.adoc

```diff
@@ -78,8 +78,12 @@ $ export ACCOUNT_KEY=`az storage account keys list -g ${RESOURCE_GROUP} --accoun
 ifdef::azure[]
 [source,terminal]
 ----
-$ export VHD_URL=`openshift-install coreos print-stream-json | jq -r '.architectures.<architecture>."rhel-coreos-extensions"."azure-disk".url'`
+$ export VHD_URL=`openshift-install coreos print-stream-json | jq -r '.architectures.<architecture>."rhel-coreos-extensions"."azure-disk".url'` 
 ----
++
+where:
+
+`<architecture>`:: Specifies the architecture, valid values include `x86_64` or `aarch64`.
```

Comment:
```suggestion

```

Reply:
Hello @kelbrown20, not sure if my suggestion clear here; Was wondering if you could not split the lines as shown in the attached image? I don't recall seeing an example in the docs where we used that formatting to express that info, but I could be totally wrong! 
![image](https://github.com/openshift/openshift-docs/assets/122639474/fbeead04-1793-4df9-9429-c107669b592a)

Reply:
Its in the doc guidelines here
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc?plain=1#L1342

It not used like super often but I think this is a good use case for it

Reply:
> Its in the doc guidelines here https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc?plain=1#L1342
> 
> It not used like super often but I think this is a good use case for it

Got it, thanks!